### DISCUSSION TITLE
Does vllm support deploy models on vuda?

### DISCUSSION THREAD

**Discussion by Gaohang0804** [vllm-project/vllm#9691] [Q&A]
I'm trying to deploy internVL2 model with vllm on k8s，I have succeed on 1 A800 80G GPU，but because the low  memory required of my model，many gpu memory is wasted.
I just wondering could deploy my model on virtualized cuda，and required lower memory on one k8s pod.

            limits:
              cpu: '15'
              memory: 16Gi
              tencent.com/vcuda-core: '100'
              tencent.com/vcuda-memory: '320'
            requests:
              cpu: '15'
              memory: 16Gi
              tencent.com/vcuda-core: '100'
              tencent.com/vcuda-memory: '320'

**Comment by Qhr77** [comment]
遇到同样问题了，在k8s上 tencent.com/vcuda-core 设成100  可以部署  改成25 这样报错 。请问你解决了吗
